src/main/kotlin/lambda/Lambda.kt

```diff
@@ -0,0 +1,11 @@
+package lambda
+
+fun sum(numbers: List<Int>, f: (number: Int) -> Boolean): Int {
+    var total = 0
+    for (number in numbers) {
+        if (f(number)) {
+            total += number
+        }
+    }
+    return total
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    return numbers.filter(f).sum()
```